### PR TITLE
config files updates

### PR DESCRIPTION
**Pull request by ebertrams**

### CODE REVIEW

File: templates/purdue/device_config.toml [hunk 1]
```diff
@@ -1,39 +1,41 @@
...
-management_uri = "unix:///var/run/iotedge/mgmt.sock"
-
-[listen]
-workload_uri = "fd://aziot-edged.workload.socket"
-management_uri = "fd://aziot-edged.mgmt.socket"
-
-[moby_runtime]
-uri = "unix:///var/run/docker.sock"
-network = "azure-iot-edge"
+## The following values will automatically be filled in when installing the configuration bundles generate by the iotedge_config_cli
```
Comment (lfitchett) [inline templates/purdue/device_config.toml:1]:
```suggestion
## The following values will automatically be filled in when installing the configuration bundles generated by the iotedge_config_cli
```

File: templates/tutorial/device_config.toml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+## The following values will automatically be filled in when installing the configuration bundles generate by the iotedge_config_cli
```
Comment (lfitchett) [inline templates/tutorial/device_config.toml:1]:
```suggestion
## The following values will automatically be filled in when installing the configuration bundles generated by the iotedge_config_cli
```

**Comment by lfitchett** [comment]
Looks good at a glance, I'm going to deploy it to my pis and double check its all working

File: templates/purdue/device_config.toml (hunk 1 above)
Comment (ebertrams) [reply templates/purdue/device_config.toml]: fixed

File: templates/tutorial/device_config.toml (hunk 2 above)
Comment (ebertrams) [reply templates/tutorial/device_config.toml]: fixed

**Comment by ebertrams** [comment]
#Sign-off from my side

**Pull request merged**